src/App.js

```diff
@@ -1,5 +1,41 @@
+// import { useEffect } from "react";
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import LogsIndex from "./Components/LogsIndex";
+import LogDetails from "./Components/LogDetails";
+import NavBar from "./Components/NavBar"
+import NewLogForm from "./Components/NewLogForm";
+import EditLogForm from "./Components/EditLogForm";
+import Home from "./Pages/Home";
+import Four0Four from "./Pages/Four0Four"
```

Comment:
Some of the import paths do not match the capitalization of your files. For instance, you have `from  "./Components/LogsIndex"` but your file name is 1`logIndex.js`